src/style.css

```diff
@@ -1,3 +1,70 @@
 body {
-  margin: 0;
+  margin: 0 auto;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin: 0 auto;
+
+  font-family: Roboto, sans-serif;
+  text-transform: uppercase;
+  font-size: 12px;
+}
+
+.div__logo {
+  display: flex;
+  align-content: center;
+}
+
+.logo {
+  margin: 10px 50px;
+  height: 40px;
+  cursor: pointer;
+}
+
+.nav {
+  display: flex;
+  margin: 0 50px;
+  height: 60px;
+}
+
+.nav__item {
+  margin-left: 20px;
+}
+
+.nav__link {
+  color: black;
+  text-decoration: none;
+  justify-items: center;
+  line-height: 60px;
+  padding: 23px 0 22px;
+  margin: 0 auto;
+  height: 60px;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+  cursor: pointer;
+}
+
+.is-active {
+  text-decoration: none;
```

Comment:
you already added text-decoration: none for nav__link why do you duplicate styles?